Tests/MongoSwiftTests/MongoCollectionTests.swift

```diff
@@ -238,7 +238,9 @@ final class MongoCollectionTests: XCTestCase {
         try coll.insertOne(["cat": nil])
         let distinct2 = try coll.distinct(fieldName: "cat", filter: [:])
         expect(distinct2).to(haveCount(3))
+        // swiftlint:disable trailing_closure
```

Comment:
is this another case like that you described in #swift?

Reply:
Yup.